cw-orch-daemon/src/channel.rs

```diff
@@ -90,6 +90,11 @@ impl GrpcChannel {
 
         Ok(successful_connections.pop().unwrap())
     }
+
+    /// Create a gRPC channel from the chain info
+    pub async fn from_chain_info(chain_info: &ChainInfoOwned) -> Result<Channel, DaemonError> {
```

Comment:
small nit: IMO should be possible to use either ChainInfo or ChainInfoOwned

Reply:
I think it's also possible to implement `TryFrom<ChainInfo> for Channel` using default `RUNTIME`

Reply:
Edit: I mis-understood. That's a good idea!
I actually rarely use that trait. Is it good practice to use it?

Reply:
> Edit: I mis-understood. That's a good idea! I actually rarely use that trait. Is it good practice to use it?

Yes from trait is usually good practice if you only need one type to get the other type. But is it bad pattern to use static runtime so mindlessly?

Reply:
I don't think we should do that implicity. It's not really straight forward and i doubt that anyone will actually try to use this type manually.